independent-publisher-connectors/Etsy/apiDefinition.swagger.json

```diff
@@ -161,7 +312,9 @@
         "responses": {
           "default": {
             "description": "default",
-            "schema": {}
+            "schema": {
+                "$ref": "#/components/schemas/Payments"

```

Comment:
@troystaylor Fund an issue while prepping your connector for production deployment - All the schema references must start with "#/definitions/". and I have not found a definition for #/components/schemas/Payments. Please fix it.

Reply:
@troystaylor Seems you have not pushed its commit to the PR. I didn't find the reference of `#/definitions/schemas/Payments"` in the swagger. And I am seeing an error saying `" System.ArgumentException: Only reference paths with 3 segments are supported."` in the swagger validation tool. Can you please fix it as soon as possible, Thanks in advance?

Reply:
Not sure why that change caused the validation to fail, but I have fixed all errors now.